frontend/src/features/admin/agencies/EditAgencyPage.tsx

```diff
@@ -0,0 +1,215 @@
+                      sendEmail: values.sendEmail,
+                      parentId: Number(values.parentId),
+                      description: values.description,
+                      rowVersion: values.rowVersion,
+                    },
+                  ),
+                );
+              } else {
+                const data = await createAgency({
+                  name: values.name,
+                  code: values.code,
+                  email: values.email,
+                  isDisabled: values.isDisabled,
+                  sendEmail: values.sendEmail,
+                  parentId: Number(values.parentId),
+                  description: values.description,
+                })(dispatch);
+                dispatch(clear(actionTypes.ADD_PARCEL));
+                history.replace(`/admin/agency/${data.id}`);
+              }
+              setSubmitting(false);
+            }}
+          >
+            {props => (
+              <Form className="agencyInfo">
+                <Label>Agency</Label>
+                <Input data-testid="agency" field="name" value={props.values.name} type="text" />
+                <Label>Short Name (Code)</Label>
+                <Input data-testid="shortName" field="code" value={props.values.code} type="text" />
+                {checkAgencies}
+                <Label>Description</Label>
+                <Input field="description" type="text" />
+                <Label>Agency e-mail address</Label>
+                <Input data-testid="email" field="email" type="text" />
+
+                <Form.Group className={'is-disabled'}>
+                  <Form.Label>
+                    Is Disabled?{' '}
+                    <TooltipIcon
+                      toolTipId="is-disabled-tooltip"
+                      toolTip={'Toggle to change account status and click save.'}
+                    />{' '}
+                  </Form.Label>
+                  <Field data-testid="isDisabled" type="checkbox" name="isDisabled" />
+                </Form.Group>
+                <Form.Group className="send-email">
+                  <Form.Label>
+                    Send e-mail{' '}
+                    <TooltipIcon
+                      toolTipId="email-tooltip"
+                      toolTip={'Toggle to change send email and click save.'}
+                    />{' '}
+                  </Form.Label>
+                  <Field type="checkbox" name="sendEmail" />
+                </Form.Group>
+
+                <div className="horizontal-line"></div>
+                <Row className="buttons">
+                  <ButtonToolbar className="cancelSave">
+                    <Button
+                      className="bg-danger mr-5"
+                      type="button"
+                      onClick={async () => setShowDelete(true)}
+                    >
+                      Delete Agency
+                    </Button>
+                    <Button className="mr-5" type="submit">
```

Comment:
this may be silently throwing some warnings in your client. If it is you'll need to e.preventDefault on the html submit event and call formiks submit method manually.